prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -1344,7 +1345,7 @@ class DatabaseAccess(val create: DSLContext) : Logging {
     }
 
     /**
-     * Inserts the provided [actionLog] using [txn] as the data context
+     * Inserts the provided [actionLog] using [txn] as the data context.
```

Comment:
Nit: if a file has nothing but whitespace or simple comment changes like this, don't update it so we can keep the PR as specific to the ticket as possible.